README.md

```diff
@@ -107,11 +107,12 @@ See [developer-facing instructions for enabling translation](https://github.com/
 
 
 - Join us on the [code4lib Slack](https://code4lib.org/irc)
+  - **#spotlight-development** - a developer-focused channel for Spotlight
   - **#blacklight** - a developer-focused channel for discussing implementation, customization, and other software concerns in the larger [Blacklight community](http://projectblacklight.org/)
   - **#spotlight-service** - a service-focused channel for people who support exhibit-builders at institutions already using Spotlight
 - Google Groups
   - [Blacklight Development Google group](https://groups.google.com/forum/#!forum/blacklight-development)
-  - [Spotlight Community Group](https://groups.google.com/forum/#!forum/spotlight-community) (equivalent to #spotlight-service)
+  - [Spotlight Community Group](https://groups.google.com/forum/#!forum/spotlight-community) (equivalent to #spotlight-service; not in active use since September 2021)
```

Comment:
Hi @mephillips-durham -- thanks for this PR to update the readme. This google group is still in use. There are posts in late 2023 and in Jan this year. I'd be happy to merge this PR if you remove the "not in active use..." statement from this line.

Reply:
Hi @corylown I have removed the remark. To be fair, at the time I created the PR, the Google group had not been in active use since September 2021, and the only postings since then have been the announcements about the Spotlight on Spotlight sessions which have received no replies. But happy to go with your suggestion in order to make sure the new slack channel is recorded.